src/ophyd_async/panda/writers/hdf_writer.py

```diff
@@ -133,9 +133,7 @@ async def open(self, multiplier: int = 1) -> Dict[str, Descriptor]:
         info = self._directory_provider()
         await asyncio.gather(
             self.hdf.file_path.set(str(info.root / info.resource_dir)),
-            self.hdf.file_name.set(
-                f"{info.prefix}.{self.panda_device.name}{info.suffix}.h5"
-            ),
+            self.hdf.file_name.set(f"{info.prefix}{info.suffix}.h5"),
```

Comment:
A note to reviewers: this change was to exclude the mandatory PandA device name from the file name. If needed, this can be added via `info.prefix` at the device instantiation step.